main.go

```diff
@@ -216,3 +266,10 @@ func getAllPackages(profiles ...*CoverProfile) []string {
 	})
 	return res
 }
+
+func renderLinkTemplate(tmpl, text string, rep *strings.Replacer) string {
+	if tmpl == "" || text == "-" {
```

Comment:
I might be missing something, but when would `text == "-"` (sorry, I'm probably having a slow day!).